templates/generate_new.html

```diff
@@ -1407,6 +1407,57 @@ <h3>Your log4shell token is active!</h3>
                                   'default':   _handleDefaultResponse
                                  };
 
+        function handleGenerateResponse( data, type ) {
+          if (data['error'] !== null){
+            if (data['error'] == '1'){
+                $('#endpoints_errors').html('Malformed request, invalid data supplied.');
+                $('#endpoints').addClass('error-outline').removeClass('success-outline');
+                $('#endpoints_errors').show();
+            } else if (data['error'] == '2'){
+                $('#memo_errors').html('Provide a memo to remind you what this token is used for.');
+                $('#memo_errors').show();
+                $('textarea[name=memo]').addClass('error-outline').removeClass('success-outline');
+            } else if (data['error'] == '3'){
+                $('#endpoints_errors').html('<p>Invalid webhook supplied, confirm you can POST to this URL with:</p><p><pre>curl -H "Content-Type: application/json" -d \'{"":""}\' URL</pre></p>');
```

Comment:
We should update this to what was in v2: `'<p>Invalid webhook supplied, confirm you can POST to this URL with:</p><p><pre>curl -H "Content-Type: application/json" -d \'{"text":"test"}\' URL</pre></p>'`